composer.json

```diff
@@ -27,6 +27,7 @@
         "symfony/console": "^4.2",
         "neos/composer-plugin": "^2.0",
         "composer/composer": "^1.9",
+        "egulias/email-validator": "^2.1",
```

Comment:
Is the requirement suppose to go both places? Wouldn't the requirement in neos/flow do the job correct?

Reply:
Yes, both places. The development collection needs those, too. See `ComposerManifestMerger.php`.